x-pack/plugins/monitoring/server/lib/standalone_clusters/has_standalone_clusters.js

```diff
@@ -52,7 +52,7 @@ export async function hasStandaloneClusters(req, indexPatterns) {
   const { callWithRequest } = req.server.plugins.elasticsearch.getCluster('monitoring');
   const response = await callWithRequest(req, 'search', params);
   if (response && response.hits) {
-    return get(response, 'hits.total.value', 0) > 0;
+    return get(response, 'hits.total', 0) > 0 || get(response, 'hits.total.value', 0) > 0;
```

Comment:
Won't this always be true in 7.x when `hits.total` is an object?

Reply:
Yes, but this code allows 6.7 to also work against 7.0

Reply:
Do we also need to set `rest_total_hits_as_int` on the request?

Reply:
We could go that route and then `hits.total` will always be an `int`. Either way, this will have to be cleaned up in the future

Reply:
I wonder if setting size to 1 and checking for hists would be better, here.

Reply:
Did this never work? The count should have always been at `hits.total`, correct? And if we want consistancy in 7.0 we should set `rest_total_hits_as_int`.